db/re/item_db_equip.yml

```diff
@@ -102334,7 +102314,6 @@ Body:
     Refineable: true
     Script: |
       .@r = getrefine();
-      bonus bMaxHP,.@r*10;
```

Comment:
This bonus is written in the aegis data but not in the description.

Actually I grep the shadow items in the aegis data : **every single one of them** have the bonus
```
bonus bBaseAtk,getrefine();
bonus bMatk,getrefine();
```
for shadow weapon and 
```
bonus bMaxHP,getrefine()*10;
```
for the others equipments, even if they have sometimes omitted the bonus in the description. I guess all the shadow eq should have the bonus of hp/atk by refine.

To resume: for me all our shadow equipment should have the extra bonus of atk/hp regardless of the description.

Reply:
Thanks